### PR TITLE
{lib}[intel/2015a] monty 0.6.4 for Python 2.7.9 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d5a77a84f9d4db2a2a51 for a full test report.

File: easybuild/easyconfigs/m/monty/monty-0.6.4-intel-2015a-Python-2.7.9-gmatteo-20150325.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+homepage = 'http://pypi.python.org/pypi/monty/'
+description = """Monty implements supplementary useful functions for Python that are not part of the standard library.
+ Examples include useful utilities like transparent support for zipped files, useful design patterns such as singleton
+ and cached_class, and many more."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://github.com/gmatteo/monty/archive/']
+sources = ['3152d41.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/monty/monty-0.6.4-intel-2015a-Python-2.7.9-gmatteo-20150325.eb]: use a proper version tag instead of a commit id

Comment (boegel) [inline easybuild/easyconfigs/m/monty/monty-0.6.4-intel-2015a-Python-2.7.9-gmatteo-20150325.eb]: I'm using `--gmatteo-20150325` as a tag for this particular version/commit, including the commit ID in the versionsuffix is meaningless

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/94992817db21517c5dfc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**